src/components/GameView/SpectatorListMenu.vue

```diff
@@ -21,12 +21,10 @@
       bg-color="surface-2"
       data-cy="spectate-list-menu"
     >
-      <v-list v-if="spectatingUsers.length > 0">
+      <v-list v-if="spectatingUsers.length > 0" color="surface-1" bg-color="surface-2">
```

Comment:
Looking at this again, why do we have 2 v-lists nested here? It looks like the inner one here is only used in order to conditionally render the list of spectators. Can we just replace this with a template tag?
```suggestion
      <template v-if="spectatingUsers.length > 0">
```

Reply:
Good call, I readjusted the html.